src/pages/home/sidebar/AllSettingsScreen.js

```diff
@@ -16,93 +15,62 @@ import ROUTES from '@src/ROUTES';
+                action: () => {
+                    Link.openOldDotLink(CONST.ADMIN_DOMAINS_URL);
                 },
-            ],
-        }),
-        [styles.accountSettingsSectionContainer],
-    );
-
-    /**
-     * Retuns JSX.Element with menu items
-     * @param {Object} data list with menu items data
-     * @returns {JSX.Element} the menu items for passed data
-     */
-    const getMenuItemsSection = useCallback(
-        (data) => (
-            <View style={[styles.pb4, styles.mh3]}>
-                {_.map(data.items, (item, index) => {
-                    const keyTitle = item.translationKey ? translate(item.translationKey) : item.title;
-
-                    return (
-                        <MenuItem
-                            key={`${keyTitle}_${index}`}
-                            wrapperStyle={styles.sectionMenuItem}
-                            title={keyTitle}
-                            icon={item.icon}
-                            shouldShowRightIcon={item.shouldShowRightIcon}
-                            iconRight={item.iconRight}
-                            disabled={isExecuting}
-                            onPress={singleExecution(() => {
-                                if (item.action) {
-                                    item.action();
-                                } else {
-                                    waitForNavigate(() => {
-                                        Navigation.navigate(item.routeName);
-                                    })();
-                                }
-                            })}
-                            shouldBlockSelection={Boolean(item.link)}
-                            focused={activeRoute && item.routeName && activeRoute.toLowerCase().replaceAll('_', '') === item.routeName.replaceAll('/', '')}
-                            isPaneMenu
-                        />
-                    );
-                })}
-            </View>
-        ),
-        [activeRoute, isExecuting, singleExecution, styles.mh3, styles.pb4, styles.sectionMenuItem, translate, waitForNavigate],
-    );
-
-    const accountMenuItems = useMemo(() => getMenuItemsSection(menuItemsData), [menuItemsData, getMenuItemsSection]);
+                shouldShowRightIcon: true,
+                iconRight: Expensicons.NewWindow,
+                link: CONST.ADMIN_DOMAINS_URL,
+            },
+        ];
+        return _.map(baseMenuItems, (item) => ({
+            key: item.translationKey,
+            title: translate(item.translationKey),
+            icon: item.icon,
+            iconRight: item.iconRight,
+            onPress: item.action,
+            shouldShowRightIcon: item.shouldShowRightIcon,
+            shouldBlockSelection: Boolean(item.link),
+            wrapperStyle: styles.sectionMenuItem,
+            isPaneMenu: true,
+            focused: item.focused,
+        }));
+    }, [styles, translate, waitForNavigate]);
 
     return (
-        <>
+        <ScreenWrapper>
```

Comment:
We need to pass `includeSafeAreaPaddingBottom={false}` and use `safeAreaPaddingBottomStyle` param and pass it to `ScrollView` just as done in the `AboutPage`. Currently we are having an extra padding / cut, diff:

<img width="457" alt="Screenshot 2024-01-09 at 1 12 05 AM" src="https://github.com/Expensify/App/assets/16493223/e47bc255-c327-43ec-8bbb-69e3bff039b6">
<img width="457" alt="Screenshot 2024-01-09 at 1 12 28 AM" src="https://github.com/Expensify/App/assets/16493223/57d063b1-2d70-48ad-8c85-44bb909e71af">